sds_data_manager/lambda_code/SDSCode/database/models.py

```diff
@@ -27,3 +22,84 @@ class UniversalSpinTable(Base):
     # TODO: create table for repointing and then make
     # a foreign key to universal_spin_table
     repointing_number = Column(Integer, nullable=False)
+
+
+class MetadataTable:
+    """Common Metadata table"""
+
+    id = Column(Integer, primary_key=True)
+    file_name = Column(String, nullable=False)
+    # TODO: Do we need instrument since these will be split up into their
+    # separate tables?
+    instrument = Column(String, nullable=False)
```

Comment:
I could see the benefit of including `instrument` if we ever want to run a query that joins the results of two or more instrument tables.